lib/queue.js

```diff
@@ -110,7 +110,9 @@ var authorizeWithSharedKey = function (method, path, query, headers) {
     var field = fields[i];
     var value = headers[field];
     if (value) {
-      stringToSign += '\n' + field + ':' + value;
+      // Convert each HTTP header to lower case and
+      // trim any white space around the colon in the header.
+      stringToSign += '\n' + field.toLowerCase() + ':' + value.trim();
```

Comment:
is whitespace in the value an issue, or only at the beginning and end?  If having whitespace inside of the string is also an issue, we should probably test for whitespace in the trimmed string and throw an exception.